CHANGELOG.md

```diff
@@ -6,6 +6,48 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ## [Unreleased]
 
+### Changed :boom:
+
+- `Maybe.all` is now type-safe and can correctly handle both array and tuple types. To support this change, it now accepts arrays or tuples directly, and the
+variadic/spread arguments to `all` have been replaced with taking an array or
+tuple directly. While `tuple` is unchanged, it is also deprecated (see below).
+
+    **Before:**
+
+    ```ts
+    import Maybe, { all, just, nothing, tuple } from 'true-myth/maybe';
```

Comment:
You're not actually using `tuple` in either of these examples; did you mean to?

Reply:
Haha, nope! Will update removing it. Thank you!